arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java

```diff
@@ -350,25 +349,38 @@ protected void applyProjection(Projection projection, FluidSearch fluidSearch, O
         }
     }
 
-    public AggregationResponse formatAggregationResult(MultiBucketsAggregation aggregation, AggregationResponse aggregationResponse, String collection) {
+    public AggregationResponse formatAggregationResult(MultiBucketsAggregation aggregation, AggregationResponse aggregationResponse, CollectionReference collection, List<Aggregation> aggregationsRequest, int i) {
```

Comment:
can `i` have a comprehensible name? (its goal is hard to understndà

Reply:
fixed